v-next/hardhat/src/internal/builtin-plugins/network-manager/request-handlers/handlers/accounts/local-accounts.ts

```diff
@@ -39,7 +42,20 @@ export class LocalAccounts extends ChainId {
     this.#initializePrivateKeys(localAccountsHexPrivateKeys);
   }
 
-  public async resolveRequest(
+  public async handle(
+    jsonRpcRequest: JsonRpcRequest,
+  ): Promise<JsonRpcRequest | JsonRpcResponse> {
+    const response = await this.#resolveRequest(jsonRpcRequest);
```

Comment:
This handler can modify a JSON-RPC request and return a JSON-RPC response. In the v2 implementation, the execution order first handled the response logic and then processed the request modification. I have retained the same approach here. [See the v2 code here](https://github.com/NomicFoundation/hardhat/blob/9ad82f53684abd6ce7e89a297233d08a8228298f/packages/hardhat-core/src/internal/core/providers/accounts.ts#L45)